apps/server/Dockerfile

```diff
@@ -15,8 +15,11 @@ RUN rm -rf /app/packages/integrations/vercel
 #  Temp solution for bun
 RUN bun install --ignore-scripts
 
+# Compile the TypeScript application
+RUN bun build --compile --sourcemap src/index.ts --outfile=app
```

Comment:
we should run this command in the `apps/server` directory 
there is no index.ts found

Reply:
Need to replace src/index.ts to apps/server !